src/shared/SwitichComponent.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+function SwitchComponent() {
```

Comment:
Correct the name of this component in the shared folder
also create a folder to store all of these related components
Profile folder for example.